js/index.js

```diff
@@ -0,0 +1,21 @@
+const today = new Date();
+const thisYear = today.getFullYear();
+
+const footer = document.querySelector('footer');
+//create copyright element and appened it to footer
+const copyright = document.createElement('p');
+copyright.innerHTML = 'Masooma Jafari ' + thisYear;
+footer.appendChild(copyright);
+
+//array of skills
+let skills = ['HTML', 'CSS', 'JavaScript', 'Vue.js', 'Mysql'];
+
+const skillSection = document.querySelector('#skills');
+//select the ul
+const skillList = skillSection.children[1];
```

Comment:
Another way you could have done this that does not depend on the `ul` being the first of the child elements, is by using the `getElementsByTagName` and passing it `ul` and then selecting the first one. However, the way you have done it here also works given that this section only contains the `ul`.

Reply:
Thank you for your review, I just add that line in my code.